cli/flox/src/commands/auth.rs

```diff
@@ -74,28 +77,124 @@ pub async fn authorize(client: BasicClient) -> Result<Credential> {
 
     debug!("Device code details: {details:#?}");
 
-    eprintln!(
-        "Please visit {} in your browser",
-        details.verification_uri().as_str()
-    );
-    eprintln!("And enter code: {}", details.user_code().secret());
+    let opener = match std::env::consts::OS {
+        "linux" => "xdg-open",
```

Comment:
if I am ssh'd into a system or don't run with a desktop what happens? I don't see an obvious check for "do I have a gui?" Asking for a me.

Reply:
i guess the open command will fail and we prompt you the url.

On linux I guess we can check if we got `DISPLAY` set and skip the `open` stuff f we have an `$SSH_TTY`..